consensus/reactor_test.go

```diff
@@ -184,6 +185,16 @@ func TestReactorWithEvidence(t *testing.T) {
 				mempoolv1.WithPreCheck(sm.TxPreCheck(state)),
 				mempoolv1.WithPostCheck(sm.TxPostCheck(state)),
 			)
+		case cfg.MempoolV2:
```

Comment:
[non-blocking] this switch statement for defining the mempool can probably be refactored into a helper. I think I've seen it 3x already.